CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Contributing to Lisk-JS
+
+First off, thanks for taking the time to contribute! :raised_hands:
+
+The following is a set of guidelines for contributing to Lisk-JS, which are hosted in the [LiskHQ Organization](https://github.com/LiskHQ) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[Help! I don't want to read this whole thing, I just have one question. :mag_right:](#help!-i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Lisk-JS Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [info@lisk.io](mailto:info@lisk.io).
+
+## Project License
+
+Every repository within LiskHQ comes with a LICENSE file. Please read it carefully before commiting your code to one of the repositories.
+
+
+## Help! I don't want to read this whole thing, I just have a question. :mag_right:
+
+Lisk is an open-source decentralized project, there are many ways and platforms to get help. These are some of them
+
+* [Discuss and Ask on Reddit](https://www.reddit.com/r/Lisk/)
+* [Lisk FAQ](https://docs.lisk.io/docs/faq)
+
+If you prefer to chat with LiskHQ and other developers directly::
+
+* [Join the LiskHQ Gitter](https://gitter.im/LiskHQ/lisk)
+* Even though Gitter is a chat service, sometimes it takes several hours for community members to respond &mdash; please be patient!
+
+## How Can I Contribute?
+
+LiskHQ uses GitHub as its sole source of truth. Everything happens here. Lightcurve employees who contribute to Lisk are expected to do so in the same way as everyone else. In other words, this document applies equally to all contributors.
+
+### `master` is unsafe
```

Comment:
master or development?

Reply:
Both actually. `development`  is changing more often than `master` but we will also update `master` with every release.

Reply:
Unsafe for what?

Reply:
Unsafe for using in production